src/components/game.tsx

```diff
@@ -4,10 +4,18 @@ import { TileStatus } from "../components/game-tile";
 import { TileState } from '../types/tile-state';
 import GameRow from './game-row';
 
-
-const CORRECT_WORD = 'CRANE';
+const CORRECT_WORD = 'THREE';
 const CORRECT_LETTERS = CORRECT_WORD.split('');
 
+const CORRECT_WORD_HASH = CORRECT_LETTERS.reduce(
+    (acc, letter) => {
+        acc[letter] ? acc[letter]++ : acc[letter] = 1;
+
+        return acc;
+    }, 
+    {} as any
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion (non-blocking):** ‏type this

```suggestion
    {} as { [letter: string]: number }
```